mkdocs.yml

```diff
@@ -327,6 +327,8 @@ nav:
       - Home Page Panels: "development/core/homepage.md"
       - User Preferences: "development/core/user-preferences.md"
       - Release Checklist: "development/core/release-checklist.md"
+    - Migration Guides:
+      - "development/migration/index.md"
```

Comment:
Maybe instead of creating a new section it might be appropriate to add a link under the `App Developer Guide`, change `Jobs Developer Guide` to a section with sub-entries, and add a link under `Jobs Developer Guide` as well?